docs/content/guides/developer/sui-101/create-coin.mdx

```diff
@@ -85,106 +87,19 @@ module examples::regcoin {
-   Version: <VERSION-NUMBER>
-   Digest: <DIGEST-HASH>
-
-   ObjectID: <OBJECT-ID>
-   Sender: <SENDER-ADDR>
-   Owner: Account Address ( <PUBLISHER-ADDRESS )
-   ObjectType: 0x2::coin::DenyCap<<PACKAGE-ID>::regcoin::REGCOIN>
-   Version: <VERSION-NUMBER>
-   Digest: <DIGEST-HASH>
-
-
-   ObjectID: <OBJECT-ID>
-   Sender: <SENDER-ADDR>
-   Owner: Account Address ( <PUBLISHER-ADDRESS )
-   ObjectType: 0x2::coin::TreasuryCap<PACKAGE-ID>::regcoin::REGCOIN>
-   Version: <VERSION-NUMBER>
-   Digest: <DIGEST-HASH>
-
-...
-```
-
-As you might have noticed, the publish action creates a `RegulatedCoinMetadata` object along with the standard `CoinMetadata` object. You don't need to explicitly call the `freeze_object` on the `RegulatedCoinMetadata` object, however, because `create_regulated_currency` automatically performs this action.
-
-The output also shows the three objects that the publisher now owns: `UpgradeCap` for [package upgrades](../../../concepts/sui-move-concepts/packages/upgrade.mdx), `TreasuryCap` for minting or burning coins, and the `DenyCap` for adding or removing addresses to or from the deny list for this coin.
-
-### Manipulate deny list
-
-For the ability to manipulate the addresses assigned to the deny list for your coin, you must add a few functions to the previous example.
-
-```move
-public fun add_addr_from_deny_list(denylist: &mut DenyList, denycap: &mut DenyCap<REGCOIN>, denyaddy: address, ctx: &mut TxContext){
-    coin::deny_list_add(denylist, denycap, denyaddy, ctx );
-}
+## Examples
 
-public fun remove_addr_from_deny_list(denylist: &mut DenyList, denycap: &mut DenyCap<REGCOIN>, denyaddy: address, ctx: &mut TxContext){
-    coin::deny_list_remove(denylist, denycap, denyaddy, ctx );
-}
-```
-
-To use these functions, you pass the `DenyList` object (`0x403`), your `DenyCap` object ID, and the address you want to either add or remove. Using the Sui CLI, you could use `sui client call` with the required information:
-
-```shell
-sui client call --function add_addr_from_deny_list --module regcoin --package <PACKAGE-ID> --args <DENY-LIST> <DENY-CAP> <ADDRESS-TO-DENY> --gas-budget <GAS-AMOUNT>
-Transaction Digest: <DIGEST-HASH>
-```
-
-The console displays the response from the network, where you can verify the DenyList object is mutated.
-
-```shell
-...
-
-MutatedObjects:
-
-  ObjectID: 0x0...403               
-  Sender: <SENDER-ADDRESS>
-  Owner: Shared
-  ObjectType: 0x2::deny_list::DenyList
-  Version: <VERSION-NUMBER>
-  Digest: <DIGEST-HASH>
-
-...
-
-```
+See the following topics for examples of some common use cases for coin and token creation.
 
-For all `Coin` functions available, see the [Sui framework documentation](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/docs/sui-framework/coin.md).
\ No newline at end of file
+- [Regulated Coin and Deny List](./create-coin/regulated.mdx): Create a regulated coin and add remove names from the deny list.
```

Comment:
nit: wording around "and add remove names from the deny list" -- is that missing a `/`?